skyportal/tests/api/test_sources.py

```diff
@@ -346,3 +346,70 @@ def test_source_notifications_unauthorized(
     assert data["message"].startswith(
         "Twilio Communication SMS API authorization error"
     )
+
+
+def test_token_user_source_summary(
```

Comment:
Check that the data we expect to not be present (obj data) are indeed not present

Reply:
done